native_client/dotnet/DeepSpeechConsole/Program.cs

```diff
@@ -60,43 +60,27 @@ static void Main(string[] args)
 
             using (IDeepSpeech sttClient = new DeepSpeech())
             {
-                var result = 1;
-                Console.WriteLine("Loading model...");
-                stopwatch.Start();
                 try
                 {
-                    result = sttClient.CreateModel(
+                    Console.WriteLine("Loading model...");
+                    stopwatch.Start();
+                    sttClient.CreateModel(
                         model ?? "output_graph.pbmm",
                         N_CEP, N_CONTEXT,
                         alphabet ?? "alphabet.txt",
                         BEAM_WIDTH);
-                }
-                catch (IOException ex)
-                {
-                    Console.WriteLine("Error loading lm.");
-                    Console.WriteLine(ex.Message);
-                }
-                stopwatch.Stop();
-                if (result == 0)
```

Comment:
Now we don't need to check for result, if an error is throw it will stop the process and display the message of the exception.

Reply:
Can we make sure the process exit code is still > 0 ?

Reply:
Yes, this doesn't change the exit code behavior. The only way I'm not sure what happens to the process exit code is when AVX2 is not supported and then the native binary crahes the whole thing without catching any exception.